src/monitoringLambda/MessageFormatter.ts

```diff
@@ -34,6 +35,18 @@ export abstract class MessageFormatter<T> {
     slackMessage.addButton('Create TopDesk ticket', 'create-topdesk-ticket', topdeskMessage.getIncident(this.priority));
     return slackMessage;
   }
+
+  /**
+   * Use the deployment environments in this project
+   * to retreive the account name
+   * @param account
+   * @returns
+   */
+  lookupAccountName(account: string) {
```

Comment:
Tof. Even in de gaten houden of naam in het bericht voldoende context is (denk het wel hoor).

Reply:
Ik bedenk me net dat dit wel een grote beperking is wanneer we meerdere configuraties hebben. Ik package het DeploymentEnvironments object (onderdeel vd cdk code) mee in de lambda. Dit betekent dus dat de sandbox in die lambda ook de configuratie gebruikt voor de main branch pakt...
Wellicht niet heel erg, maar we kunnen ook een object pakken obv de branch name (klein beetje refactoring), zal ik dat er nog in bouwen?